OfficeIMO.Tests/OfficeIMO.Tests.csproj

```diff
@@ -1,8 +1,8 @@
 <Project Sdk="Microsoft.NET.Sdk">
 
     <PropertyGroup>
-        <TargetFrameworks Condition=" '$([MSBuild]::IsOsPlatform(`Windows`))' ">net472;net48;netcoreapp3.1;net6.0;net7.0</TargetFrameworks>
-        <TargetFrameworks Condition=" '$([MSBuild]::IsOsPlatform(`OSX`))' Or '$([MSBuild]::IsOsPlatform(`Linux`))' ">net6.0;net7.0</TargetFrameworks>
+        <TargetFrameworks>net6.0;net7.0</TargetFrameworks>
+        <TargetFrameworks Condition=" '$([MSBuild]::IsOsPlatform(`Windows`))' ">net472;net48;netcoreapp3.1;$(TargetFrameworks)</TargetFrameworks>
```

Comment:
I believe net core 3.1 also works on OSX/Linux so should be in all.

Reply:
I understand, and you're probably right. We can ignore 3.1, but then we need to remove it everywhere, not just in SLN.